testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -35,6 +35,7 @@
     SUPPORTS_AGGREGATION_PUSHDOWN_COVARIANCE(false),
     SUPPORTS_AGGREGATION_PUSHDOWN_CORRELATION(false),
     SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION(false),
+    SUPPORTS_AGGREGATION_PUSHDOWN_COUNT_DISTINCT(false),
```

Comment:
Do we need a new connector behavior at all?

can we assume `count(DISTINCT)` to be pushed down whenever `SUPPORTS_AGGREGATION_PUSHDOWN && SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY`?

Reply:
then in this commit would need to enable `ImplementCountDistinct` for much more than PSQL.
@alexjo2144 do you happen to know if feature is enablable for all relevant DBs? 
For sake of making turnover on this PR shorter I would prefer to limit it just to PSQL here. And send followup PR which possibly removes the SUPPORTS_AGGREGATION_PUSHDOWN_COUNT_DISTINCT flag.

Reply:
> do you happen to know if feature is enablable for all relevant DBs?

I didn't go through them all, that's mostly why I separated `ImplementCountDistinct`, to limit the change to only connectors I tested (PSQL)